src/utils/fromPairs.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Unless we find an existing module, let's just replace the single occurrence of this utility with:
```js
var result = pairs.reduce((result, [key, value]) => {
  result[key] = value;
  return result;
}, {});
```